harvester/core/processors/pipeline/http.py

```diff
@@ -0,0 +1,78 @@
+from django.db.models import Q
+from django.contrib.contenttypes.models import ContentType
+
+from datagrowth.configuration import create_config
+from datagrowth.resources.http.tasks import send
+from datagrowth.processors import Processor
+
+from core.processors.pipeline.base import PipelineProcessor
+
+
+class HttpPipelineProcessor(PipelineProcessor):
+
+    def filter_documents(self, queryset):
+        depends_on = self.config.pipeline_depends_on
+        pipeline_phase = self.config.pipeline_phase
+        filters = Q(**{f"pipeline__{depends_on}__success": True}) | Q(**{f"pipeline__{pipeline_phase}__success": False})
+        return queryset.filter(filters)
+
+    def process_batch(self, batch):
+
+        config = create_config("http_resource", self.config.retrieve_data)
+        app_label, resource_model = config.resource.split(".")
+        resource_type = ContentType.objects.get_by_natural_key(app_label, resource_model)
+
+        updates = []
+        creates = []
+        for process_result in batch.processresult_set.all():
+            args, kwargs = process_result.document.output(config.args, config.kwargs)
+            successes, fails = send(*args, **kwargs, config=config, method=config.method)
+            results = successes + fails
+            if not len(results):
+                continue
+            result_id = results.pop(0)
+            process_result.result_type = resource_type
+            process_result.result_id = result_id
+            updates.append(process_result)
+            for result_id in results:
+                # TODO: create docs here where necessary
```

Comment:
kan dit later?

Reply:
Ja, deze is meer voor Datagrowth. Momenteel hebben we niet echt een case hiervoor